changelog/_unreleased/2025-01-27-refactor-country-language-snipptes.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Refactor country language snipptes
+issue: NEXT-40285
+---
+# Administration
+* Changed `src/Administration/Resources/app/administration/src/module/sw-settings-country/snippet/en-GB.json` and `src/Administration/Resources/app/administration/src/module/sw-settings-country/snippet/de-DE.json` to clarify the meaning
```

Comment:
imo such changelogs can completely be omitted. there is nothing relevant for internal or externals, right?